File: docs/code-examples/box3d_simple.rs
//! Log a single 3D box.

fn main() -> Result<(), Box<dyn std::error::Error>> {
    let rec = rerun::RecordingStreamBuilder::new("rerun_example_box3d").spawn()?;

    rec.log(
        "simple",
        &rerun::Boxes3D::from_half_sizes([(2.0, 2.0, 1.0)]),
    )?;

    Ok(())
}

File: docs/code-examples/box3d_simple.cpp
// Log a single 3D box.

#include <rerun.hpp>

int main() {
    auto rec = rerun::RecordingStream("rerun_example_box3d_simple");
    rec.spawn().throw_on_failure();

    rec.log("simple", rerun::Boxes3D::from_half_sizes({{2.f, 2.f, 1.0f}}));
}
